src/generated/data/tz.json

```diff
@@ -1 +1,1978 @@
"Europe/Athens"},{"codes":["RS","BA","HR","ME","MK","SI"],"name":"Europe/Belgrade"},{"codes":["DE","DK","NO","SE","SJ"],"name":"Europe/Berlin"},{"codes":["BE","LU","NL"],"name":"Europe/Brussels"},{"codes":["RO"],"name":"Europe/Bucharest"},{"codes":["HU"],"name":"Europe/Budapest"},{"codes":["MD"],"name":"Europe/Chisinau"},{"codes":["IE"],"name":"Europe/Dublin"},{"codes":["GI"],"name":"Europe/Gibraltar"},{"codes":["FI","AX"],"name":"Europe/Helsinki"},{"codes":["TR"],"name":"Europe/Istanbul"},{"codes":["RU"],"name":"Europe/Kaliningrad"},{"codes":["RU"],"name":"Europe/Kirov"},{"codes":["UA"],"name":"Europe/Kyiv"},{"codes":["PT"],"name":"Europe/Lisbon"},{"codes":["GB","GG","IM","JE"],"name":"Europe/London"},{"codes":["ES"],"name":"Europe/Madrid"},{"codes":["MT"],"name":"Europe/Malta"},{"codes":["BY"],"name":"Europe/Minsk"},{"codes":["RU"],"name":"Europe/Moscow"},{"codes":["FR","MC"],"name":"Europe/Paris"},{"codes":["CZ","SK"],"name":"Europe/Prague"},{"codes":["LV"],"name":"Europe/Riga"},{"codes":["IT","SM","VA"],"name":"Europe/Rome"},{"codes":["RU"],"name":"Europe/Samara"},{"codes":["RU"],"name":"Europe/Saratov"},{"codes":["RU","UA"],"name":"Europe/Simferopol"},{"codes":["BG"],"name":"Europe/Sofia"},{"codes":["EE"],"name":"Europe/Tallinn"},{"codes":["AL"],"name":"Europe/Tirane"},{"codes":["RU"],"name":"Europe/Ulyanovsk"},{"codes":["AT"],"name":"Europe/Vienna"},{"codes":["LT"],"name":"Europe/Vilnius"},{"codes":["RU"],"name":"Europe/Volgograd"},{"codes":["PL"],"name":"Europe/Warsaw"},{"codes":["CH","DE","LI"],"name":"Europe/Zurich"},{"codes":["IO"],"name":"Indian/Chagos"},{"codes":["MV","TF"],"name":"Indian/Maldives"},{"codes":["MU"],"name":"Indian/Mauritius"},{"codes":["WS"],"name":"Pacific/Apia"},{"codes":["NZ","AQ"],"name":"Pacific/Auckland"},{"codes":["PG"],"name":"Pacific/Bougainville"},{"codes":["NZ"],"name":"Pacific/Chatham"},{"codes":["CL"],"name":"Pacific/Easter"},{"codes":["VU"],"name":"Pacific/Efate"},{"codes":["TK"],"name":"Pacific/Fakaofo"},{"codes":["FJ"],"name":"Pacific/Fiji"},{"codes":["EC"],"name":"Pacific/Galapagos"},{"codes":["PF"],"name":"Pacific/Gambier"},{"codes":["SB","FM"],"name":"Pacific/Guadalcanal"},{"codes":["GU","MP"],"name":"Pacific/Guam"},{"codes":["US"],"name":"Pacific/Honolulu"},{"codes":["KI"],"name":"Pacific/Kanton"},{"codes":["KI"],"name":"Pacific/Kiritimati"},{"codes":["FM"],"name":"Pacific/Kosrae"},{"codes":["MH"],"name":"Pacific/Kwajalein"},{"codes":["PF"],"name":"Pacific/Marquesas"},{"codes":["NR"],"name":"Pacific/Nauru"},{"codes":["NU"],"name":"Pacific/Niue"},{"codes":["NF"],"name":"Pacific/Norfolk"},{"codes":["NC"],"name":"Pacific/Noumea"},{"codes":["AS","UM"],"name":"Pacific/Pago Pago"},{"codes":["PW"],"name":"Pacific/Palau"},{"codes":["PN"],"name":"Pacific/Pitcairn"},{"codes":["PG","AQ","FM"],"name":"Pacific/Port Moresby"},{"codes":["CK"],"name":"Pacific/Rarotonga"},{"codes":["PF"],"name":"Pacific/Tahiti"},{"codes":["KI","MH","TV","UM","WF"],"name":"Pacific/Tarawa"},{"codes":["TO"],"name":"Pacific/Tongatapu"}]
```

Comment:
Undo prettier formatting 
(write file with script, update prettierignore)

Same for the others

Reply:
It's already in prettierignore, that's an intended change, as I described in the PR description, it's to make changes more visible.